waffle/tests/test_waffle.py

```diff
@@ -334,6 +334,35 @@ def test_pluggable_model(self):
         self.assertEqual(b'off', response.content)
         assert 'dwf_myflag' not in response.cookies
 
+    @override_settings(WAFFLE_CREATE_MISSING_FLAGS=True)
+    @override_settings(WAFFLE_FLAG_DEFAULT=False)
+    def test_flag_created_dynamically_default_false(self):
+        flag_model = waffle.get_waffle_flag_model()
```

Comment:
Extract all of this into a helper method (e.g. `assert_flag_dynamically_created(expected_value)`).